resources/js/styles.js

```diff
@@ -15,7 +15,31 @@ $( document ).ready( function() {
             }
         });
    });
-   $('.delete').click(function(){
+   var counter = 2;
+    $('#add-text-subject').click(function(e){
+        e.preventDefault();
+        var textNew = $(document.createElement('div')).attr({"id":"box_group"+counter});
+        textNew.after().html(`<div class="form-group">
+                <label>Tên chủ đề</label>
```

Comment:
what is this? đoạn này là gì đây? sao xử lý luôn trong js này

Reply:
Dạ đoạn js này em dùng để thêm một cái div á anh.. Vì một course sẽ có nhiều subject. Nên khi nhập thì mình sẽ tạo ra một cái div mới rồi nhập cái subject tiếp theo trong một cái view luôn ạ.